Tests/LinuxMain.swift

```diff
@@ -126,7 +126,8 @@ extension SharedMiddlewareTransformersTests {
     ("testRedirectUnrelatedHosts", testRedirectUnrelatedHosts),
     ("testRequireHerokuHttps", testRequireHerokuHttps),
     ("testRequireHttps", testRequireHttps),
-    ("testRequestLogger", testRequestLogger)
+    ("testRequestLogger", testRequestLogger),
+    ("testBasiAuthValidationIsCaseInsensitive", testBasiAuthValidationIsCaseInsensitive)
```

Comment:
Will need to regenerate or update this, too:
```suggestion
    ("testBasicAuthValidationIsCaseInsensitive", testBasicAuthValidationIsCaseInsensitive)
```